extensions/amp-story/1.0/amp-story.js

```diff
@@ -347,6 +368,8 @@ export class AmpStory extends AMP.BaseElement {
       ? new AmpStoryViewerMessagingHandler(this.win, this.viewer_)
       : null;
 
+    this.indexOfLastPageToPreview_ = this.calculateIndexOfLastPageToPreview_();
```

Comment:
Is there a reason to calculate this in `buildCallback` instead of the constructor? (eg: something not being initialized in the constructor yet).

Usually I'd argue that it's more deterministic to initialize the variable to the correct value (a number) instead of calculating it later (cause it can be null or number), but that might be my opinion and not general consensus 😂.

Reply:
I would also rather calculate in the constructor, but the viewer is only initialized in `buildCallback`, and it's necessary for getting the param values. I thought about moving the viewer initialization to the constructor, but I was afraid I'd unintentionally break something lol